recipes/cfitsio/all/conanfile.py

```diff
@@ -0,0 +1,109 @@
+import glob
+import os
+
+from conans import ConanFile, CMake, tools
+
+class CfitsioConan(ConanFile):
+    name = "cfitsio"
+    description = "C library for reading and writing data files in FITS " \
+                  "(Flexible Image Transport System) data format"
+    license = "ISC"
+    topics = ("conan", "cfitsio", "fits", "image", "nasa", "astronomy", "astrophysics", "space")
+    homepage = "https://heasarc.gsfc.nasa.gov/fitsio/"
+    url = "https://github.com/conan-io/conan-center-index"
+    exports_sources = ["CMakeLists.txt", "patches/**"]
+    generators = "cmake"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "threadsafe": [True, False],
+        "simd_intrinsics": [None, "sse2", "ssse3"],
```

Comment:
this should be removed for all arches except **x86** and **x86_64**

Reply:
Done, good catch.